### DISCUSSION TITLE
Is it possible to check the current url in handlebar / publii ?

### DISCUSSION THREAD

**Discussion by hubyhuby** [GetPublii/Publii#1370] [Help] [answered]
I need to insert a specific tracker for google add on a page that has the url "/contact".
Is there a way to  detect the url path of the current page generated ?
for example 
  {{#if URL as "/contact"}}
MY SCRIPT

For now I am doing it in javascript like this:

<script>

  if (window.location.href.includes("rdv")) {

    console.log("rdv");
     
    gtag('event', 'conversion', {'send_to': 'AW-TRACKING HERE '});
}
</script>

**Comment by bobmitro** [comment, marked as answer]
Yeap, for example, you can do this via a post ID using the checkIf helper:  https://getpublii.com/dev/checkif-helper/

`{{#post}}
    {{#checkIf id '==' 10}}
          your script here...
    {{/checkIf}}
{{/post}}`

**Reply by hubyhuby** [reply]
WHaouuuuu this is awsome !! Thks a lot @bobmitro

**Comment by dziudek** [comment]
@hubyhuby - it is not your case, but it is also worth to know that in some cases the `{{#is}}` helper can be useful: https://getpublii.com/dev/is-helper/ - but it is more for page types than for specific subpages :)
